Docs/aso-setup-v2.md

```diff
@@ -67,7 +67,9 @@ Note: You need to [install yq](https://mikefarah.gitbook.io/yq/) for these scrip
 
 ## Postgres Flexible Server
 
-- Run [postgres-setup.sh](../bin/v2/postgres-setup.sh) with your namespace and app name.
+- Run [aso-postgres-server.sh](../bin/v2/aso-postgres-server.sh) with your namespace and app name.
    ```bash
```

Comment:
Being a bit pedantic but can we make it `add-postgres-server`?

Reply:
yep i think that's a good idea let me modify :)